control-plane/connect-inject/container_init.go

```diff
@@ -66,6 +68,9 @@ type initContainerCommandData struct {
 	// TProxyExcludeUIDs is a list of additional user IDs to exclude from traffic redirection via
 	// the consul connect redirect-traffic command.
 	TProxyExcludeUIDs []string
+
+	// ConsulDNSIP is the IP of the Consul DNS Service.
+	ConsulDNSIP string
```

Comment:
```suggestion
	// ConsulDNSIP is the IP of the Consul DNS Service.
	ConsulDNSClusterIP string
```

Maybe use cluster ip just so it's crystal?